core/block/simple/dataview/dataview.go

```diff
@@ -221,6 +221,33 @@ func (l *Dataview) FillSmartIds(ids []string) []string {
 		if view.DefaultTemplateId != "" {
 			ids = append(ids, view.DefaultTemplateId)
 		}
+		ids = append(ids, getIdsFromFilters(view.Filters)...)
+	}
+
+	return ids
+}
+
+func getIdsFromFilters(filters []*model.BlockContentDataviewFilter) (ids []string) {
+	for _, filter := range filters {
+		if filter.Format != model.RelationFormat_object &&
```

Comment:
do we need file objects here? (and special RelationKeyCoverId)

Reply:
It seems our clients do not support filters by relations with File format, so we cannot meet such objects in dataview block